.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/TimePeriodParserChs.cs

```diff
@@ -76,6 +83,76 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refDate)
             return null;
         }
 
+        private DateTimeResolutionResult ParseNight(string text, DateObject referenceTime)

```

Comment:
This should be renamed to ParseTimeOfDay, no? Maybe also in other languages.

Reply:
Ok, updated.